### DISCUSSION TITLE
Mystery 1px margin on navbar on large screen

### DISCUSSION THREAD

**Discussion by oliversalzburg** [squidfunk/mkdocs-material#3472] [General]
On a large screen (2000px+), when I view the official docs, like <https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/>, in Chrome, there is a mystery 1px margin above the navbar when scrolling down.

![image](https://user-images.githubusercontent.com/1658949/149795691-91692d02-6cb3-417f-afec-778f4f5093f1.png)

I didn't instantly see the root cause for this. I also do not have this behavior with my own documentation. Just wanted to bring it up.

**Comment by squidfunk** [comment]
Thanks for reporting. Could you please provide some more details, i.e., which browser, version and operating system you're using? I've seen this behavior on iOS Safari on several pages, so it might be a browser bug.

**Reply by oliversalzburg** [reply]
```
Google Chrome | 97.0.4692.71 (Official Build) (64-bit) (cohort: Stable)
Revision | adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}
OS | Windows 10 Version 21H2 (Build 19044.1415)
```

**Reply by oliversalzburg** [reply]
Unsurprisingly, also reproduces in Edge 97.0.1072.62.
Does not reproduce in Firefox Developer Edition 97.0b4.

**Reply by squidfunk** [reply]
Thanks for the update. Since `.md-header` already has `top: 0` defined, I'm guessing it's related to fixed positioning and browser issues. The fact that it reproduces in Chrome, Edge and iOS Safari sounds like it's related to Blink/WebKit. Nothing we can fix.
